onnxruntime/python/tools/transformers/models/stable_diffusion/README.md

```diff
@@ -74,90 +74,116 @@ Below is an example to optimize Stable Diffusion 1.5 in Linux. For Windows OS, p
+Skip this step if you use onnxruntime-gpu package from official releases.
 
 To try latest optimizations, you can install [ort-nightly-gpu](https://aiinfra.visualstudio.com/PublicPackages/_artifacts/feed/ORT-Nightly/PyPI/ort-nightly-gpu/) package like the following:
 
 ```
-pip uninstall onnxruntime-gpu
-pip install ort-nightly-gpu -i https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/ORT-Nightly/pypi/simple/
+pip3 uninstall onnxruntime-gpu
+pip3 install ort-nightly-gpu -i https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/ORT-Nightly/pypi/simple/
 ```
 
 ### Setup Environment (ROCm)
 
-It is recommended that the users should run the model with ROCm 5.4 or newer and Python 3.9. Note that Windows is not
-supported for ROCm at the moment.
+It is recommended that the users run the model with ROCm 5.4 or newer and Python 3.8, 3.9 or 3.10.
+Note that Windows is not supported for ROCm at the moment.
 
 ```
-conda create -n py39 python=3.9
-conda activate py39
-pip install -r requirements.txt
-pip install -r requirements-rocm.txt
+conda create -n py38 python=3.8
+conda activate py38
+wget https://repo.radeon.com/rocm/manylinux/rocm-rel-5.4/torch-1.12.1%2Brocm5.4-cp38-cp38-linux_x86_64.whl
+pip3 install torch-1.12.1+rocm5.4-cp38-cp38-linux_x86_64.whl
+pip3 install -r requirements-rocm.txt
 ```
 
-AMD GPU version of torch build can be installed from [AMD Radeon repo](https://repo.radeon.com/rocm/manylinux/rocm-rel-5.4/),
-user need to download the whl file and `pip install <file.whl>` manually. Or directly from https://download.pytorch.org/whl/rocm5.4.2 via
+AMD GPU version of PyTorch can be installed from [pytorch.org](https://pytorch.org/get-started/locally/) or [AMD Radeon repo](https://repo.radeon.com/rocm/manylinux/rocm-rel-5.4/).
+
+#### Install onnxruntime-rocm
+
+We install [ROCm 5.4.2](https://docs.amd.com/bundle/ROCm-Installation-Guide-v5.4.2/page/How_to_Install_ROCm.html) since that is also used in PyTorch 2.0 ROCm package.
+
+Here is an example to build onnxruntime from source with Rocm 5.4.2, and install the wheel.
 ```
-pip install torch==2.0.0 --index-url https://download.pytorch.org/whl/rocm5.4.2
+sudo apt-get update
+sudo apt-get install -y --no-install-recommends \
+        wget \
+        zip \
+        ca-certificates \
+        build-essential \
+        curl \
+        libcurl4-openssl-dev \
+        libssl-dev \
+        python3-dev
+pip3 install numpy packaging "wheel>=0.35.1"
+wget --quiet https://github.com/Kitware/CMake/releases/download/v3.26.3/cmake-3.26.3-linux-x86_64.tar.gz
+tar zxf cmake-3.26.3-linux-x86_64.tar.gz
+export PATH=${PWD}/cmake-3.26.3-linux-x86_64/bin:${PATH}
+
+git clone https://github.com/microsoft/onnxruntime
+cd onnxruntime
+sh build.sh --config Release --cmake_extra_defines onnxruntime_USE_COMPOSABLE_KERNEL=ON  \
+            --use_rocm --rocm_home /opt/rocm --rocm_version 5.4.2 --build_wheel --allow_running_as_root
```

Comment:
Lets make it a bit shorter:

sh build.sh --config Release --use_rocm --rocm_home /opt/rocm --rocm_version 5.4.2 --build_wheel